api/edgeproto/cloudlet.proto

```diff
@@ -516,8 +522,8 @@ message Cloudlet {
   PlatformConfig config = 21 [(gogoproto.nullable) = false, (protogen.backend) = true];
   // Optional resource to restagtbl key map key values = [gpu, nas, nic]
   map<string, edgeproto.ResTagTableKey> res_tag_map = 22 [(protogen.backend) = true, (protogen.refers_to) = "ResTagTable"];
-  // Secrets required to access cloudlet, will be saved in encrypted storage
-  map<string, string> access_vars = 23;
+  // Secrets required to access cloudlet, will be saved in encrypted storage, and cleared on the cloudlet object
+  map<string, string> access_vars = 23 [(protogen.backend) = true, (protogen.hidetag) = "nocmp"];
```

Comment:
Doesn't adding `(protogen.backend) = true` make this not configurable?

Reply:
No, it just removes it from comparisons in unit/e2e tests. "noconfig" makes it not configurable by the user.

Reply:
Yeah, I see the corresponding code in mex.go. So comment in `protogen.proto` is just incorrect:
```
  // Backend defines a field that is filled in by the back-end,
  // not by the user.
  bool backend = 51013;
```

Reply:
Yeah, it's a little misleading. I think at one point it was meant to be as you said, only configurable by the back-end, as opposed to how it's treated now, which is *may* be configured by the back-end.